pkgs/build-support/wrapper-common/utils.bash

```diff
@@ -146,7 +146,7 @@ checkLinkType() {
 # When building static-pie executables we cannot have rpath
 # set. At least glibc requires rpath to be empty
 filterRpathFlags() {
-    local linkType=$1 ret="" i
+    local linkType=$1 ret i
```

Comment:
> would you mind adding a declaration on a separate line with -a for ret. I think it makes it clearer that the variable is intended to be used as an array.

I could do, but are you sure you want me to change that? It would be redundant; you don't need to declare indexed arrays in Bash.

>  I have no idea what ret stands for, perhaps return?

Yeah, it's a strange part of Bash. It never returns anything other than an exit code, but based on how other languages work I think a lot of people think of the standard output as the "return value".

Reply:
It's really a matter of personal taste. I personally think it's indicative of the intent. I'm happy to merge without it.

Reply:
Agreed, it's mostly taste. Thanks for being flexible though, much appreciated!